coral-component-taglist/src/scripts/TagList.js

```diff
@@ -310,9 +314,46 @@ class TagList extends BaseFormField(BaseComponent(HTMLElement)) {
     // Cleans the tag from TagList specific values
     detachedItem.removeAttribute('role');
     detachedItem.removeAttribute('tabindex');
-    detachedItem._host = undefined;
-    
-    if (this._itemToFocusAfterDelete) {
+    const parentElement = this.parentElement;
+    if (this.items.length === 0 && parentElement) {
+        // If all tags are removed, call focus method on parent element 
+        if (typeof parentElement.focus === 'function') {
+            parentElement.focus();
+    }
+
+    // clear orphaned ::before and ::after from .coral3-TagList:empty TagList which should default to display: none
+        this.innerHTML = '';
+
+        var self = this;
+
+    window.requestAnimationFrame(() => {
+          // if the parentElement did not receive focus or move focus to some other element
+          if (document.activeElement.tagName === 'BODY') {
+            // make the TagList focusable
+            self.tabIndex = -1;
+            self.classList.add('u-coral-screenReaderOnly');
+            self.style.outline = '0';
+            self.innerHTML = ' ';
+            var onBlurFocusManagement = function onBlurFocusManagement() {
+              self.removeAttribute('tabindex');
+              self.classList.remove('u-coral-screenReaderOnly');
+              self.style.outline = '';
+              self.innerHTML = '';
+              self._vent.off('blur.focusManagement');
+            };
+            self._vent.on('blur.focusManagement', null, onBlurFocusManagement); 
+            Coral.commons.nextFrame(function() {
```

Comment:
`Coral.commons` should be imported as `commons` ES6: https://github.com/adobe/coral-spectrum/pull/118/files#diff-2d4442dc115c547d389cafa6622d7184R17